src/annotator/components/test/NotebookModal-test.js

```diff
@@ -0,0 +1,107 @@
+import NotebookModal from '../NotebookModal';
+import { EventBus } from '../../util/emitter';
+import { mount } from 'enzyme';
+import { act } from 'preact/test-utils';
+
+describe('NotebookModal', () => {
+  let container;
+  let components;
+  let eventBus;
+
+  const createComponent = config => {
+    const component = mount(
+      <NotebookModal
+        eventBus={eventBus}
+        config={{ notebookAppUrl: '/notebook', ...config }}
+      />,
+      {
+        attachTo: container,
+      }
+    );
+    components.push(component);
+    return component;
+  };
+
+  beforeEach(() => {
+    container = document.createElement('div');
+    components = [];
+    document.body.appendChild(container);
+    eventBus = new EventBus();
+  });
+
+  afterEach(() => {
+    components.forEach(component => component.unmount());
+    container.remove();
+  });
+
+  it('hides modal on first render', () => {
+    const wrapper = createComponent();
+    const outer = wrapper.find('.Notebook__outer');
+
+    assert.isTrue(outer.hasClass('is-hidden'));
+    assert.equal(getComputedStyle(wrapper.getDOMNode()).display, 'none');
+
+    // It also contains the iframe, although doesn't have a group.
+    const iframe = wrapper.find('iframe');
+    assert.isTrue(iframe.exists());
+    assert.equal(
+      iframe.prop('src'),
+      `/notebook#config=${encodeURIComponent('{"group":""}')}`
+    );
+  });
+
+  it('shows modal on openNotebook event', () => {
```

Comment:
```suggestion
  it('shows modal on "openNotebook" event', () => {
```

I find it helpful when reading test descriptions if event names are quoted.